### ISSUE TITLE
Show IPv6 addresses in network scan.

### ISSUE THREAD

**Issue by rokejulianlockhart** [herzhenr/simple-wake-on-lan#7]
![Screenshot_20230822-002605](https://github.com/herzhenr/simple-wake-on-lan/assets/42837531/50ca68aa-d668-49cf-bbe0-64f00b5ec79a

Dependent upon https://github.com/herzhenr/simple-wake-on-lan/issues/4#issue-1860306490.

**Comment by herzhenr** [comment]
See Issue #4. The networking part of the app would have to be reworked and I am not sure how to scan a whole subnet for `ipv6` addresses. It also comes down to the [wake on lan](https://pub.dev/packages/wake_on_lan) dart package supporting `ipv6`.

**Comment by Paul-Etienne** [comment]
Apparently, [wake on lan](https://pub.dev/packages/wake_on_lan) now supprots `IPv6`.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/herzhenr/simple-wake-on-lan/issues/7#issuecomment-2198586477

@Paul-Etienne, indeed, per https://github.com/JagandeepBrar/package-wake-on-lan/issues/4#issuecomment-1698421055 and https://github.com/JagandeepBrar/package-wake-on-lan/blob/b724565df10aff00fb253b4096d38540bbd552b9/README.md?plain=1#L75C1-L75C16.
